University-advisor/SqlDriver.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data.SQLite;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace University_advisor
+{
+    class SqlDriver
+    {
+        private static SQLiteConnection connect()
+        {
+            try
+            {
+                SQLiteConnection dbConnection = new SQLiteConnection("Data Source=Database.sqlite;Version=3;");
+                dbConnection.Open();
+                return dbConnection;
+            }
+            catch (SQLiteException e)
+            {
+                Console.WriteLine(e.StackTrace);
+                return null;
+            }
+        }
+
+        public static void exec(string sql)
+        {
+            SQLiteConnection dbConnection = connect();
+            if(dbConnection == null)
+            {
+                Console.WriteLine("Failed to connect to database. Aborting query");
+                return;
+            }
+            try
+            {
+                SQLiteCommand command = new SQLiteCommand(sql, dbConnection);
+                command.ExecuteNonQuery();
+            }
+            catch (SQLiteException e)
+            {
+                Console.WriteLine(e.StackTrace);
+                return;
+            }
+            dbConnection.Close();
+        }
+
+        public static ArrayList fetch(string sql)
+        {
+            SQLiteConnection dbConnection = connect();
+            if (dbConnection == null)
+            {
+                Console.WriteLine("Failed to connect to database. Aborting query");
+                return null;
+            }
+            try
+            {
+                ArrayList al = new ArrayList();
```

Comment:
I think it is better to use generic `List<T>` instead of `ArrayList`

Reply:
Is EntityAttributesModel a type already implemented in the sql library or do you suggest making a new class?

Reply:
I suggest making a new class with properties that we are going to use. You can put it into Models directory.

Reply:
I don't really understand how this would correlate with fetching data from our database. Are you implying we should make our application with MVC pattern?

Reply:
I guess that can be discussed with others but I believe that MVC pattern would simplify developing process and testing. What is more, I think implementing dependency injection principle would be easier in such scenario. So, leave how it is for now and we will discuss this topic during the next meeting.